notebooks/README.md

```diff
@@ -1,10 +1,77 @@
 # Example notebooks
 
-These are example notebooks to showcase cuxfilter. Related Datasets can be download using the link below:
+These are example notebooks to showcase cuxfilter.
 
-- Download the mortgage dataset from [here](https://docs.rapids.ai/datasets/mortgage-viz-data)
+## TRY CUXFILTER NOTEBOOKS ONLINE
 
-- Nyc taxi dataset from [here](https://drive.google.com/file/d/1mTvl66VLzHwQJPcgnGBdmZTNEdNp1tYo/view?usp=sharing)
+[<img src="https://img.shields.io/badge/-Setup Studio Lab Environment-gray.svg">](#amazon-sagemaker-studio-lab) [<img src="https://img.shields.io/badge/-Setup Colab Environment-gray.svg">](#google-colab)
 
+1. Mortgage_example.ipynb
 
-- Auto dataset from [here](https://drive.google.com/file/d/1jxySYJ9e32hI8PQ5QPr9_xrsu37N5fOM/view?usp=sharing)
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/Mortgage_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/Mortgage_example.ipynb) 
+
+2. NYC_taxi_example.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/NYC_taxi_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/NYC_taxi_example.ipynb)
+
+3. auto_accidents_example.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/auto_accidents_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/auto_accidents_example.ipynb)
+
+4. graphs.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb) 
+    [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb)
+</br>
+
+
+## Setup Remote Environments
+### Amazon Sagemaker Studio Lab
+
+[Amazon SageMaker Studio Lab](https://studiolab.sagemaker.aws/faq) is a free online web application for learning and experimenting with data science and machine learning using Jupyter notebooks and is free to use.
```

Comment:
Amazon SageMaker Studio Lab is a free ML development environment that provides the compute, storage (up to 15GB), and security —all at no cost (currently). This includes GPU notebook instances.

Reply:
fixed